### ISSUE TITLE
Docker mainnet unable to connect

### ISSUE THREAD

**Issue by yancyribbens** [lightningnetwork/lnd#2337]
### Background

docker does not work on mainnet using the current configuration

### Steps to reproduce

export NETWORK="mainner"
docker-compose run --name alice lnd_btc

### Expected behaviour

stops while trying to connect to btcd backed fee estimator

```
Creating btcd ... done
2018-12-17 22:31:20.020 [WRN] LTND: open /root/.lnd/lnd.conf: no such file or directory
2018-12-17 22:31:20.020 [INF] LTND: Version: 0.5.1-beta commit=v0.5.1-beta-75-g89f81bd528ec57c7144439583dab8254439c828e-dirty, build=production, logging=default
2018-12-17 22:31:20.020 [INF] LTND: Active chain: Bitcoin (network=mainnet)
2018-12-17 22:31:20.024 [INF] CHDB: Checking for schema update: latest_version=7, db_version=7
2018-12-17 22:31:20.024 [INF] RPCS: Generating TLS certificates...
2018-12-17 22:31:20.032 [INF] RPCS: Done generating TLS certificates
2018-12-17 22:31:20.086 [INF] LTND: Primary chain is set to: bitcoin
2018-12-17 22:31:20.088 [INF] LTND: Initializing btcd backed fee estimator```

### Actual behavior

**Comment by wpaulino** [comment]
Seems related to #2336.

**Comment by yancyribbens** [comment]
@wpaulino I think this is a different issue.  There is no problem connecting to testnet as in #2336.

**Comment by wpaulino** [comment]
Assuming this is creating a new `btcd` node, you'd have to give it time to sync.

**Comment by Roasbeef** [comment]
This would mean that maybe it isn't able to actually connect to the `btcd` instance. I'd double check any authentication parameters, and that you can also reach one container from the other.

**Comment by yancyribbens** [comment]
@Roasbeef this is caused by btcd being invoked with a network flag `--mainnet`.  In order to run btcd in mainnet there should be no network defined:

incorrect which is what `start-btcd.sh` composes:
```bash-4.4# btcd --mainnet --debuglevel=info ----rpcuser=devuser --rpcpass=devpass --datadir=/data --logdir=/data --rpccert=/rpc/rpc.cert --rpckey=/rpc/rpc.key --rpclisten=0.0.0.0 --txindex                                                  
unknown flag `mainnet'
Use btcd -h to show usage```

correct invocation of mainnet
```bash-4.4# btcd --debuglevel=info ----rpcuser=devuser --rpcpass=devpass --datadir=/data --logdir=/data --rpccert=/rpc/rpc.cert --rpckey=/rpc/rpc.key --rpclisten=0.0.0.0 --txindex                                                  
unknown flag `mainnet'
Use btcd -h to show usage```

**Comment by yancyribbens** [comment]
commit from a different issue seems to fix this: https://github.com/lightningnetwork/lnd/commit/05a449e99b68e70f67950a639886f4c4d6d878dd

**Comment by Roasbeef** [comment]
Fixed by #2246
